src/ui/render.ts

```diff
@@ -511,16 +574,20 @@ export default class UI {
     });
   }
 
+  showMenu() {
+    console.log("TODO: Render menu here");
+  }
+
   handleMouseDown(e: MouseEvent, colIndex: number, rowIndex: number) {
-    if (e.button === 0) {
-      this.selectedCellsContainer.selectionStart =
-        (colIndex != null || undefined) && (rowIndex != null || undefined)
-          ? {
-              row: rowIndex,
-              column: colIndex,
-            }
-          : null;
-    }
+    if (e.button !== 0) return;
+    this.hideContextMenu();
```

Comment:
Since this is in `handleMouseDown`, clicking elements such as the formula bar (or just clicking empty space on the page) doesn't hide the context menu. From a usability perspective, shouldn't the context menu be hidden on all mouse clicks, regardless of what element it's performed on?

Reply:
Agreed that it would be ideal, but could not figure out a way to do it. Suggestions are welcome :)